src/js/controller/settings/exportimage/ImageExportController.js

```diff
@@ -7,7 +7,15 @@
     this.gifExportController = new ns.GifExportController(piskelController);
   };
 
+  pskl.utils.inherit(ns.ImageExportController, pskl.controller.settings.AbstractSettingController);
+
   ns.ImageExportController.prototype.init = function () {
+    // Output Scaling Factor
+    var scalingFactorInput = document.querySelector('.scaling-factor-input');
+    scalingFactorInput.value = pskl.UserSettings.get(pskl.UserSettings.EXPORT_SCALING);
+    this.addEventListener(scalingFactorInput, 'change', this.onScalingFactorChange_);
```

Comment:
It would be nice to have a listener on the 'input' event as well : 
```javascript
    this.addEventListener(scalingFactorInput, 'change', this.onScalingFactorChange_);
    this.addEventListener(scalingFactorInput, 'input', this.onScalingFactorChange_);
```
On browsers that support the 'input' event properly on range inputs, this will fire everytime the value is updated, so you will see the number change in real time.